app/frontend/controllers/tab_controller.js

```diff
@@ -784,6 +784,47 @@ class TabController {
     }
   }
 
+  async updateTabTitleAfterLocaleChange() {
+    for (let i = 0; i < this.metaTabs.length; i++) {
+      const currentMetaTab = this.metaTabs[i];
+      let tabTitle = "";
+      switch (currentMetaTab.getTextType()) {
+        case 'book':
+          tabTitle = await i18nHelper.getSwordTranslation(currentMetaTab.getBook());
+          break;
+        case 'search_results':
+          tabTitle = this.getSearchTabTitle(currentMetaTab.getSearchTerm());
+          break;
+        case 'tagged_verses': {
+            const refTitle = getRefFromTabTitle(currentMetaTab.taggedVersesTitle);
+            if (refTitle) {
+              tabTitle += `${refTitle} &ndash; `;
+            }
+            if (platformHelper.isElectron()) {
+              tabTitle += `${i18n.t('tags.verses-tagged-with')} `;
+            }
+            tabTitle += `<i>${currentMetaTab.tagTitleList}</i>`;
+          }
+          break;  
+        case 'xrefs': {
+            tabTitle = `${getRefFromTabTitle(currentMetaTab.xrefTitle)} &ndash; ${i18n.t("general.module-xrefs")}`;
```

Comment:
Any ideas how to localize `currentMetaTab.xrefTitle`. 
`currentMetaTab.getBook()` is empty for xrefs tab type

Reply:
> > > For some reason the selectMenu widget has dark formatting (even though not in darkmode):
> > 
> > 
> > It looks better now, but the color is still different from the Bible translation menu, shouldn't it be the same?
> 
> Somehow default jQueryUI styles despaired when I attached the dropdown menu to the component element. So I applied custom styles inside the component. I think those colors are better on the dialog background vs having them the same as dialog. I don't think it messes up the consistency. Those colors are already in the interface and that select-box is quite unique in terms it's popup over popup. But let me know if you have other concerns.

It's not a blocking point for me and your explanations make sense!

Reply:
> Any ideas how to localize `currentMetaTab.xrefTitle`.
> `currentMetaTab.getBook()` is empty for xrefs tab type

I'll look into this tomorrow.